### ISSUE TITLE
Check values of @meiversion

### ISSUE THREAD

**Issue by claudius108** [Edirom/MerMEId#207]
Check what is about values like ```meiversion="2013"``` in files ```mei_to_html.xsl```, ```view_score.xq```, and ```bibl_reference_types.xml```..

From MEI Guidelines, the legit values are: ```3.0.0```, ```4.0.0```, ```4.0.1```, ```5.0```, ```5.0+anyStart```, ```5.0+basic"```, ```5.0+CMN```, ```5.0+Mensural```, and ```5.0+Neumes```.

**Comment by peterstadler** [comment]
Yes, that looks like some very old legacy code. It's probably save to modify `mei_to_html.xsl` and `view_score.xq`, with `bibl_reference_types.xml` I'm a little bit unsure since this is just pseudo MEI (i.e. not a valid MEI file) anyways.
